### ISSUE TITLE
Fix incorrect pole for Rigify

### ISSUE THREAD

**Issue by OliverJPost** [OliverJPost/HumGen3D#108]
**Describe the bug**
The arm does not move realistically in IK mode for Rigify

**Comment by Kif11** [comment]
Yep, I'm experiencing that as well. Also when I try to switch to Poll Target mode for Arm IK controls the gizmos do not show up.
